docs/sources/articles/baseimages.md

```diff
@@ -41,22 +41,21 @@ GitHub Repo:
  - [Debian / Ubuntu](
    https://github.com/docker/docker/blob/master/contrib/mkimage-debootstrap.sh)
 
-## Creating a simple base image using `scratch`
+## Creating a simple base image using scratch
```

Comment:
Christy, thank you for your contribution.  I think some of the sentences can be tightened up also, we should refer to the hub not the registry.  Also, the `FROM` syntax is not special for `scratch` as far as I know so we don't want to imply that.

----> https://gist.github.com/moxiegirl/082970bd8c684a314f71

## Creating a simple base image using scratch

You can use Docker's reserve, minimal image, `scratch`, as a starting point for building containers. Using the `scratch` "image" signals to the build process that you want the next command in the `Dockerfile` to be the first filesystem layer in your image.

While `scratch` appears in the Docker's repository on the hub, you can't pull, run, or tag any image with the `scratch` name. Instead, you can refer to it in your Dockerfile. For example, to create a minimal container using  `scratch`:

    FROM scratch
    ADD hello /
    CMD ["/hello"]

This example creates the `hello-world` image used in the tutorials!
If you want to test it out, you can clone [the image repo](https://github.com/docker-library/hello-world).